Makefile

```diff
@@ -12,6 +12,15 @@ SRC_BAREBOX = $(MAKEFILE_DIR)/external/barebox
 BUILD_BSP = $(MAKEFILE_DIR)/build/b-$(BSP)
 LIBBSD_BUILDSET = $(MAKEFILE_DIR)/src/libbsd.ini
 
+UNAME := $(shell uname -s)
+ifeq ($(UNAME),Darwin)
+	NUMCORE = $(shell sysctl -n hw.ncpu)
```

Comment:
That should work on FreeBSD too (jut tested it on a FreeBSD 12). So maybe that case should include `FreeBSD`.